inst/shiny/DiagnosticsExplorer/R/OrphanConceptsModule.R

```diff
@@ -97,7 +97,7 @@ orphanConceptsModule <- function(id,
         data <- data %>%
           dplyr::filter(is.na(.data$standardConcept) |
                           (
-                            !is.na(.data$standardConcept) && .data$standardConcept != "S"
+                            any(!is.na(.data$standardConcept), .data$standardConcept != "S")
```

Comment:
I assume the change from a logical "AND" to any (which is OR) fixes the issue with it loading?